source/reference/configuration/file-setting-cli-option-mapping.txt

```diff
@@ -0,0 +1,29 @@
+.. _c2c-config-setting-option-map:
+
+Configuration File Settings and Command-Line Option Mapping
```

Comment:
Since the primary use of this page is the Command-Line reference, can you update the title to be "Command-Line Options".